src/_locales/es/messages.json

```diff
@@ -354,7 +354,7 @@
         "description": ""
     },
     "tooltip_block": {
-        "message": "Mover el control deslizante a la izquierda para bloquear un dominio",
+        "message": "Mueva el control deslizante a la izquierda para bloquear un dominio",
         "description": ""
     },
     "options_enable_dnt_checkbox": {
```

Comment:
Should we switch the "" quotes here to «» like in the `non_tracker_tip` message?

Reply:
Yes